packages/components/src/components/domainproperties/issues/IssuesPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,221 @@
+
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Comment sort direction" required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        componentClass="select"
+                        id="commentSortDirection"
+                        onChange={onInputChange}
+                        value={value ? value : ''}
+                    >
+                        <option key="Oldest first" value="ASC">
+                            Oldest first
+                        </option>
+                        <option key="Newest first" value="DESC">
+                            Newest first
+                        </option>
+                    </FormControl>
+                </Col>
+
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class AssignedToGroupInput extends React.PureComponent<SecurityUserGroupProps, any> {
+    onChange = (name: string, formValue: any, selected: Principal, ref: any): any => {
+        if (selected && this.props.onSelect) {
+            this.props.onSelect(selected);
+        }
+    };
+
+    render() {
+        const { coreGroups } = this.props;
+        const name = 'addGroupAssignment';
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Assigned to list comes from" required={false} />
+                </Col>
+                <Col xs={9} lg={8}>
+                    <SelectInput
+                        name={name}
+                        options={coreGroups.toArray()}
+                        placeholder=""
+                        inputClass="col-xs-12"
+                        valueKey="userId"
+                        labelKey="displayName"
+                        onChange={this.onChange}
+                        formsy={false}
+                        showLabel={false}
+                        multiple={false}
+                        required={false}
+                    />
+                </Col>
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class DefaultUserAssignmentInput extends React.PureComponent<SecurityUserGroupProps, any> {
+    onChange = (name: string, formValue: any, selected: Principal, ref: any): any => {
+        if (selected && this.props.onSelect) {
+            this.props.onSelect(selected);
+        }
+    };
+
+    render() {
+        const { coreUsers } = this.props;
+        const name = 'defaultUserAssignment';
+
+        function getFilteredCoreUsers(groupId : any)
+        {
+            let filteredCoreUser = coreUsers.filter(coreUser => {
+                return coreUser.groupId === groupId;
+            });
+            return filteredCoreUser.toArray().length > 0 ? filteredCoreUser.toArray() : coreUsers.toArray();
+        }
```

Comment:
I'd move this function out of the render function since having it hear means that it is re-defined each time the render method is called for this component. If you move it as a sibling to render, then you can call it via `this.getFilteredCoreUsers()`.

Reply:
now moved